providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -269,6 +275,27 @@ def execute_async(
         # try and remove it from the QUEUED state while we process it
         self.last_handled[key] = time.time()
 
+    def queue_workload(self, workload: workloads.ExecuteTask, session: Session | None) -> None:
+        ti = workload.ti
+        self.queued_tasks[ti.key] = workload
+
+    def _process_workloads(self, workloads: list[workloads.All]) -> None:
+        # Airflow V3 version
+        for w in workloads:
+            # TODO: AIP-72 remove once actual tokens are working.
+            if not w.token:  # type: ignore[union-attr]
+                w.token = "placeholder-token"  # type: ignore[union-attr]
```

Comment:
Why is this needed? I don't think this is in local or celer exec is it?

Lets tie the comment to #45107

Reply:
Oh yeah, its overriden as empty string anyways. I will remove it

Reply:
Added comment for it too